Source/interfaces/IDisplayInfo.h

```diff
@@ -60,6 +81,26 @@ namespace Exchange {
             virtual void Updated() = 0;
         };
 
+        struct EXTERNAL IHdrIterator : virtual public Core::IUnknown {
+            enum { ID = ID_CONNECTION_PROPERTIES_HDR };
+            virtual ~IHdrIterator() {}
+
+            enum HdrFormats : uint8_t {
+                HDR_NONE = 0,
+                HDR_10,
+                HDR_HLG,
+                HDR_DOLBYVISION,
+                HDR_TECHNICOLORPRIME,
+                HDR_10PLUS,
+                HDR_UNKNOWN
+            };
+
+            virtual bool IsValid() const = 0;
+            virtual bool Next() = 0;
+            virtual void Reset() = 0;
+            virtual HdrFormats HdrFormat() const = 0;
+        };
+
```

Comment:
@pwielders, Based on your comments https://github.com/rdkcentral/Thunder/pull/214#discussion_r467864644 ,
I have introduced HDR iterator to relay the HDR formats over this interface.